### ISSUE TITLE
New post email notification: Image (Tiled Gallery Block) is not responsive, but it stretches instead

### ISSUE THREAD

**Issue by AtrumGeost** [Automattic/wp-calypso#47188]
#### Steps to reproduce

1. Follow the test site
2. Publish a blog post with a portrait sized image  (tiled gallery block)
3. Check the email notification on mobile

Additional information: the user said this is a recent issue form the last week (so it could be related to a recent change)

#### What I expected

For the image to be displayed in a responsive way, just like on the website:

![Screen Shot 2020-11-06 at 08 23 10](https://user-images.githubusercontent.com/3696121/98370934-5bb03080-2009-11eb-8bd1-49079e3f99d9.png)

#### What happened instead

The image is stretched:

![IMG_EA8F8326AC6C-1](https://user-images.githubusercontent.com/3696121/98371141-b0ec4200-2009-11eb-81f9-898b156f2fa9.jpeg)

#### Browser / OS version

The latest version of Safari on iPhone Xs running iOs 14.2.

The same happens if you use an email client like Apple Mail.

#### Context / Source

#user-report

They asked to be notified when the issue is fixed:

25453505-hc

**Comment by kriskorn** [comment]
User back in chat asking for updates: #25454782-hc

**Comment by cathymcbride** [comment]
@AtrumGeost I'm not reproing this. Can you add more details to the issue please? Is there a particular site or image I should use?

**Comment by AtrumGeost** [comment]
Hi @cathymcbride! Thanks for checking this :)

I checked my test post here:

pb330k-45-p2

And since I only copied the content directly from the user's post I missed to notice that they used a **Tiled Gallery block**. If you use an Image block it will work fine. But when running a new test with the tiled gallery block I was able to make the issue even more noticeable both on desktop and mobile:

**Original post:**

pb330k-4b-p2

![Screen Shot 2020-11-10 at 12 49 17](https://user-images.githubusercontent.com/3696121/98711583-28480b80-2353-11eb-97db-7e6bb3422e57.png)

**Desktop email:**

![Screen Shot 2020-11-10 at 12 48 46](https://user-images.githubusercontent.com/3696121/98711612-326a0a00-2353-11eb-8bd9-58872850d01f.png)

**Mobile email:**

![IMG_B01E616F968E-1](https://user-images.githubusercontent.com/3696121/98711685-50d00580-2353-11eb-9e39-46d93a4391ed.jpeg)

**Comment by obenland** [comment]
@ianstewart Is there a chance you might have any helpful tips as to how to debug and fix block styles in notification emails?

**Comment by ianstewart** [comment]
No, but perhaps @mikeshelton1503 might have some specific email tips?

**Comment by Agepeuve** [comment]
Another report at 3741306-zen.

**Comment by mikeshelton1503** [comment]
No specific idea here but you can try using Litmus It's a email testing platform that will show the email various email clients. We have still have an account I believe.

On a quick search this thread might be a similar issue: https://litmus.com/community/discussions/5220-image-stretched-vertically-responsive-email-question

**Comment by NoHopeRadio** [comment]
Reporting another incident at hc-28269170

**Comment by AnjanaV93** [comment]
Another report in 7411138-zen
